src/Entity/Factory/ShippingMethodExampleFactory.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity\Factory;
+
+use App\Entity\Shipping\ShippingMethod;
+use Sylius\Bundle\CoreBundle\Fixture\Factory\ExampleFactoryInterface;
+use Sylius\Bundle\CoreBundle\Fixture\Factory\ShippingMethodExampleFactory as BaseShippingMethodExampleFactory;
+use Sylius\Component\Channel\Repository\ChannelRepositoryInterface;
+use Sylius\Component\Core\Model\ShippingMethodInterface;
+use Sylius\Component\Resource\Factory\FactoryInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+final class ShippingMethodExampleFactory extends BaseShippingMethodExampleFactory implements ExampleFactoryInterface
+{
+    /** @var FactoryInterface */
+    private $shippingMethodFactory;
+
+    /** @var RepositoryInterface */
+    private $zoneRepository;
+
+    /** @var RepositoryInterface */
+    private $shippingCategoryRepository;
+
+    /** @var RepositoryInterface */
+    private $taxCategoryRepository;
+
+    /** @var RepositoryInterface */
+    private $localeRepository;
+
+    /** @var ChannelRepositoryInterface */
+    private $channelRepository;
+
+    /** @var \Faker\Generator */
+    private $faker;
+
+    /** @var OptionsResolver */
+    private $optionsResolver;
+
+    public function __construct(
+        FactoryInterface $shippingMethodFactory,
+        RepositoryInterface $zoneRepository,
+        RepositoryInterface $shippingCategoryRepository,
+        RepositoryInterface $localeRepository,
+        ChannelRepositoryInterface $channelRepository,
+        ?RepositoryInterface $taxCategoryRepository = null
+    ) {
+        parent::__construct(
+            $shippingMethodFactory,
+            $zoneRepository,
+            $shippingCategoryRepository,
+            $localeRepository,
+            $channelRepository,
+            $taxCategoryRepository
+        );
+    }
+
+    public function create(array $options = []): ShippingMethodInterface
+    {
+        /** @var ShippingMethod $shippingMethod */
+        $shippingMethod = parent::create($options);
+
+        if(!isset($options['deliveryConditions'])) {
+            return $shippingMethod;
+        }
+        $shippingMethod->setDeliveryConditions($options['deliveryConditions']);
```

Comment:
If I see correctly, `deliveryConditions` is a translatable field, so you should probably do sth like:
```
    foreach ($this->getLocales() as $localeCode) {
        $shippingMethod->setCurrentLocale($localeCode);
        $shippingMethod->setFallbackLocale($localeCode);

        $shippingMethod->setDeliveryConditions($options['deliveryConditions']);
    }
```